src/spaceinvaders/boss.js

```diff
@@ -0,0 +1,36 @@
+import { Player } from './player.js';
+
+export class Boss extends Player {
```

Comment:
I think the name of the class will be "bonus" is more appropriate, plus it can't be a player subclass, they don't have the same behavior